Makefile

```diff
@@ -24,4 +24,4 @@ Sources/OrderedSet/OrderedSet.swift:
 Tests/OrderedSetTests/OrderedSetTests.swift:
 	@echo $@
 	@echo "// From $(TEST_URL)" > $@
-	@curl -L $(TEST_URL) >> $@
+	@curl -L $(TEST_URL) | sed 's/TSCBasic/OrderedSet_swift/g' >> $@
```

Comment:
```suggestion
	@curl -L $(TEST_URL) | sed 's/TSCBasic/OrderedSet/g' >> $@
```

Reply:
I don't think we can, because an [OrderedSet pod](https://github.com/Weebly/OrderedSet) already exists, right?

Reply:
The pod name remains as `OrderedSet.swift`, but the target name can be `OrderedSet`.